odl/contrib/datasets/ct/mayo.py

```diff
@@ -126,74 +139,83 @@ def load_projections(folder, indices=None):
     # For unknown reasons, mayo does not include the tag
     # "TableFeedPerRotation", which is what we want.
     # Instead we manually compute the pitch
-    pitch = ((datasets[-1].DetectorFocalCenterAxialPosition -
-              datasets[0].DetectorFocalCenterAxialPosition) /
-             ((np.max(angles) - np.min(angles)) / (2 * np.pi)))
+    table_dist = datasets[-1].DetectorFocalCenterAxialPosition - \
+                 datasets[0].DetectorFocalCenterAxialPosition
```

Comment:
Please use parentheses instead of backslash to break a line.
```suggestion
    table_dist = (datasets[-1].DetectorFocalCenterAxialPosition
                  - datasets[0].DetectorFocalCenterAxialPosition)
```

👍 for breaking the logic into two parts, makes it much clearer what's going on.